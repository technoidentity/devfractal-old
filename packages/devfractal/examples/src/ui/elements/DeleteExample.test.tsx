import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Delete } from './DeleteExample'

test('Delete Example', () => {
  const component: ReactTestRenderer = renderer.create(<Delete />)
  expect(component.toJSON()).toMatchSnapshot()
})
