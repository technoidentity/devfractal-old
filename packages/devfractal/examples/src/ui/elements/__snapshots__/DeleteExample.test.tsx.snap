// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Dynamic Delete(
      click Me!
      )
    </h1>
    <a
      className="delete"
      onClick={[Function]}
    />
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Default Delete
    </h1>
    <a
      className="delete"
    />
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Sizes
    </h1>
    <div
      className="box"
    >
      <a
        className="delete is-small"
      />
      <a
        className="delete"
      />
      <a
        className="delete is-medium"
      />
      <a
        className="delete is-large"
      />
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Combinations
    </h1>
    <div
      className="box"
    >
      <div
        className="tags has-addons"
      >
        <span
          className="tag is-success"
        >
          Hello World
          <a
            className="delete"
          />
        </span>
      </div>
      <div
        className="notification is-danger"
      >
        <a
          className="delete is-large"
          onClick={[Function]}
        />
        Lorem ipsum dolor sit amet, consectetur adipiscing elit lorem ipsum dolor sit amet, consectetur adipiscing elit
      </div>
      <article
        className="message is-info"
      >
        <div
          className="message-header"
        >
          Info
          <a
            className="delete is-medium"
          />
        </div>
        <div
          className="message-body"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque risus mi, tempus quis placerat ut, porta nec nulla. Vestibulum rhoncus ac ex sit amet fringilla. Nullam gravida purus diam, et dictum felis venenatis efficitur. Aenean ac eleifend lacus, in mollis lectus. Donec sodales, arcu et sollicitudin porttitor, tortor urna tempor ligula, id porttitor mi magna a neque. Donec dui urna, vehicula et sem eget, facilisis sodales sem.
        </div>
      </article>
    </div>
  </section>,
]
`;
